packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.5+3
+
+* Migrating styleFrom API to new version.
```

Comment:
This needs to follow https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style, so it should be "Migrates".

Reply:
Thanks for reminding me of this. I have fixed it on the latest commit.